File: VB/ManageUsersOnLogon.Module/Module.vb
Imports System
Imports Security.Extensions
Imports DevExpress.ExpressApp
Imports System.Collections.Generic
Imports ManageUsersOnLogon.Module.DatabaseUpdate

Namespace ManageUsersOnLogon.Module
    Public NotInheritable Partial Class ManageUsersOnLogonModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
        End Sub
        Shared Sub New()
            SecurityExtensionsModule.CreateSecuritySystemUser = AddressOf Updater.CreateUser
        End Sub
    End Class
End Namespace

File: CS/ManageUsersOnLogon.Module/Module.cs
using System;
using Security.Extensions;
using DevExpress.ExpressApp;
using System.Collections.Generic;
using ManageUsersOnLogon.Module.DatabaseUpdate;

namespace ManageUsersOnLogon.Module {
    public sealed partial class ManageUsersOnLogonModule : ModuleBase {
        public ManageUsersOnLogonModule() {
            InitializeComponent();
        }
        static ManageUsersOnLogonModule() {
            SecurityExtensionsModule.CreateSecuritySystemUser = Updater.CreateUser;
        }
    }
}
